modules/gapi/misc/python/python_bridge.hpp

```diff
@@ -11,6 +11,7 @@
 #include <opencv2/gapi/garg.hpp>
 #include <opencv2/gapi/gopaque.hpp>
 #include <opencv2/gapi/render/render_types.hpp> // Prim
+#include <opencv2/gapi/ot_tracking_status.hpp> // TrackingStatus
```

Comment:
Should it be just a part of `ot.hpp`?

Reply:
+1

Reply:
It couldn't be that time, as was included in `gcommon.hpp`. Now, it is removed from `gcommon,hpp` and returned back to `ot.hpp`